chore: migrate to Vue 3

ShGKme opened the pull request.

ShGKme committed:
refactor: migrate to vue 3

Signed-off-by: Grigorii K. Shartsev <me@shgk.me>

nickvergessen commented:
> vue-select in @nextcloud/vueis not from our fork in v9...

Was this fixed in the meantime?

ShGKme commented:
The problem has been fixed in `v9.7.0`
- https://github.com/nextcloud-libraries/nextcloud-vue/pull/8423

ShGKme committed:
build(deps): bump to vue 3 family

```
npm un @nextcloud/vue vue vuex vue-loader vue-template-compiler
@nextcloud/webpack-vue-config
npm i @nextcloud/vue vue
npm i -D @nextcloud/webpack-vue-config
```

Signed-off-by: Grigorii K. Shartsev <me@shgk.me>

ShGKme committed:
fix: POST config value as string

Signed-off-by: Grigorii K. Shartsev <me@shgk.me>

mejo- reviewed (approved):
Thanks a lot ❤️
